docs/contributing.rst

```diff
@@ -93,6 +93,12 @@ Alternatively, there is a testing script at ``scripts/test.sh`` which executes a
 
 Travis CI even runs some additional tests (including ``solc-js`` and testing third party Solidity frameworks) that require compiling the Emscripten target.
 
+.. note ::
+
+While any version of ``cpp-ethereum`` should be usable, this cannot be guaranteed, and it is suggested to use the same version as used by the Solidity continuous integration tests are doing.
```

Comment:
`as used` ... `are doing`

Reply:
Should we explicitly mention that certain commit ranges of ``cpp-ethereum`` are known to fail due to bugs?

Reply:
I wouldn't go as far as that, but at least mention the hash we are using, which is supposed to work.